lib/sassc/script/value_conversion.rb

```diff
@@ -0,0 +1,44 @@
+module SassC
+  module Script
+    module ValueConversion
+      class Unsupported < StandardError; end
```

Comment:
How do you feel about creating a base Exception class in https://github.com/bolandrm/sassc-ruby/blob/master/lib/sassc/error.rb that the rest inherit from so that it’s easier to rescue from “A SassC error”?